Made some fixes to the create estimate page.

Chelsea002 opened the pull request:
I worked on the create estimate page (Evaluate Estimation) where i fixed the 'How much (in hours) do you charge per hour', by eliminating it as directed by the team lead. I also Fixed (coded using HTML5 and CSS3) the Navbar so it looks like the general navigation bar as displayed on the figma design. I finally fixed the NEXT button, at the footer to its appropriate size.

Hosted link: http://create-estimate.000webhostapp.com/

![fixed](https://user-images.githubusercontent.com/54754405/67389919-78f79580-f593-11e9-8f4d-21daa1966305.PNG)

beardedprince commented:
Good job bro. But check the drop down on pages like project.html

Adopt such dropdown for your sidebar... that's the only fix here

Chelsea002 committed:
Made some fixes to the create estimate page.

I worked on the create estimate page (Evaluation Estimation) where i fixed the 'How much (in hours) do you charge per hour', by eliminating it as directed by the team lead. I also Fixed (coded using HTML5 and CSS3) the Navbar so it looks like the general navigation bar as displayed on the figma design. I finally fixed the NEXT button, at the footer to its appropriate size.
